core/src/main/scala/se/lu/nateko/cp/meta/core/data/DataObject.scala

```diff
@@ -150,6 +150,9 @@ case class DocObject(
 	doi: Option[String],
 	fileName: String,
 	size: Option[Long],
+	title: Option[String],
```

Comment:
`references: References` already have a place for authors and title. Why not use those instead of creating fields with duplicate content?